src/Avalonia.Themes.Fluent/Controls/SplitButton.xaml

```diff
@@ -0,0 +1,205 @@
+    <Setter Property="Template">
+      <Setter.Value>
+        <ControlTemplate>
+          <Grid>
+            <Grid.ColumnDefinitions>
+              <ColumnDefinition Width="*" />
+              <ColumnDefinition Width="Auto" />
+            </Grid.ColumnDefinitions>
+
+            <Button x:Name="PART_PrimaryButton"
+                    Grid.Column="0"
+                    MinWidth="{DynamicResource SplitButtonPrimaryButtonSize}"
+                    Foreground="{TemplateBinding Foreground}"
+                    Background="{TemplateBinding Background}"
+                    BorderThickness="{TemplateBinding BorderThickness}"
+                    BorderBrush="{TemplateBinding BorderBrush}"
+                    Content="{TemplateBinding Content}"
+                    ContentTemplate="{TemplateBinding ContentTemplate}"
+                    Command="{TemplateBinding Command}"
+                    CommandParameter="{TemplateBinding CommandParameter}"
+                    FontFamily="{TemplateBinding FontFamily}"
+                    FontSize="{TemplateBinding FontSize}"
+                    FontWeight="{TemplateBinding FontWeight}"
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Stretch"
+                    HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+                    VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+                    Padding="{TemplateBinding Padding}"
+                    Focusable="False"
+                    KeyboardNavigation.IsTabStop="False" />
+
+            <Button x:Name="PART_SecondaryButton"
+                    Grid.Column="1"
+                    MinWidth="{DynamicResource SplitButtonSecondaryButtonSize}"
+                    BorderBrush="{TemplateBinding BorderBrush}"
+                    Foreground="{TemplateBinding Foreground}"
+                    Background="{TemplateBinding Background}"
+                    BorderThickness="{TemplateBinding BorderThickness}"
+                    HorizontalContentAlignment="Right"
+                    VerticalContentAlignment="Center"
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Stretch"
+                    Focusable="False"
+                    KeyboardNavigation.IsTabStop="False" />
+          </Grid>
+        </ControlTemplate>
+      </Setter.Value>
+    </Setter>
+  </Style>
+
+  <!-- Default overridable styles -->
+  <Style Selector="SplitButton /template/ Button#PART_PrimaryButton">
+    <Setter Property="CornerRadius" Value="{TemplateBinding CornerRadius, Converter={StaticResource LeftCornerRadiusFilterConverter}}" />
+  </Style>
+  <Style Selector="SplitButton /template/ Button#PART_SecondaryButton">
+    <Setter Property="CornerRadius" Value="{TemplateBinding CornerRadius, Converter={StaticResource RightCornerRadiusFilterConverter}}" />
+    <Setter Property="Padding" Value="9,0" />
```

Comment:
The Padding here was specifically set to "0,0,10,0" on purpose.

 1. "0,0,10,0" Matches with ComboBox
 2. 10 on the right is from the math based on 32 width: 32 - 12 = 20; 20 / 2 = 10. This will center a size 12 glyph in the center of a 32px button. Elsewhere I have seen 9 used instead of 10 but I'm not sure why -- perhaps layout rounding.
 3. This really should be ONLY on the right in case the button size ever changes. That is the Fluent standard. I don't think setting this to left/top only is a good idea.
     - As mentioned, it should be position from the right, not the left
     - If ever the height changes it will no longer be centered.

Reply:
Visually, I can see the 1px error in glyph position using 9. This definitely needs to change to 10. Then in Fluent v2 where there is a 1px border it would change to 9. Alternatively, we could just center it horizontally/vertically. That wouldn't be an issue since the secondary button is no longer stretched to full width for touch devices.

Reply:
Yes, centering it will be the best solution.